xmtp_mls/src/groups/members.rs

```diff
@@ -0,0 +1,90 @@
+use std::collections::HashMap;
+
+use xmtp_proto::api_client::{XmtpApiClient, XmtpMlsClient};
+
+use crate::identity::Identity;
+
+use super::{GroupError, MlsGroup};
+
+#[derive(Debug, Clone)]
+pub struct GroupMember {
+    pub wallet_address: String,
+    pub installation_ids: Vec<Vec<u8>>,
+}
+
+impl<'c, ApiClient> MlsGroup<'c, ApiClient>
+where
+    ApiClient: XmtpApiClient + XmtpMlsClient,
+{
+    // Load the member list for the group from the DB, merging together multiple installations into a single entry
+    pub fn members(&self) -> Result<Vec<GroupMember>, GroupError> {
+        let openmls_group =
+            self.load_mls_group(&self.client.mls_provider(&mut self.client.store.conn()?))?;
+
+        let member_map: HashMap<String, GroupMember> = openmls_group
+            .members()
+            .filter_map(|member| {
+                Identity::get_validated_account_address(
+                    member.credential.identity(),
+                    &member.signature_key,
+                )
+                .ok()
+                .map(|wallet_address| (wallet_address, member.signature_key.clone()))
+            })
+            .fold(
+                HashMap::new(),
+                |mut acc, (wallet_address, signature_key)| {
+                    acc.entry(wallet_address.clone())
+                        .and_modify(|e| e.installation_ids.push(signature_key.clone()))
```

Comment:
didn't know about this method!

Reply:
I finally found a Rust thing @insipx doesn't know about! Gonna go call my mom